coordinator/src/services/spine-communication.ts

```diff
@@ -70,18 +70,36 @@ export class SandboxRequestHandler implements RequestHandler {
     }
   }
 
-  async poll(path: string): Promise<SpineResponse<string | OperationOutcome>> {
+  async poll(path: string): Promise<SpineResponse<OperationOutcome>> {
     if (path === this.parentPrescriptionPollingId) {
-      //TODO - add realistic response
+      const parentPrescriptionOperationOutcome = {
```

Comment:
instead of `const parent...Outcome = {...} as OperationOutcome`, think its more standard to do `const parent...Outcome: OperationOutcome = {...}`